src/ode_solver/implicit_ode_solver.cpp

```diff
@@ -1,5 +1,6 @@
 #include "implicit_ode_solver.h"
```

Comment:
Since I see that the changes are trivial, this file is a sign that you added all the files at some point through:
git add . or git add -A or git add -u

My suggestion is to add relevant files manually with the purpose of the commit description in mind.

Fine for this PR, but careful in the future as it might add a lot of files you don't want it to.

Reply:
Yes, this was a mistake in the first commit of the branch. I've been more careful with adding files in subsequent commits!